ida/data.yml

```
@@ -7856,85 +7857,88 @@ classes:
       71: GetDESCRI
       72: GetMJICWSP
       73: GetPERFORM
-      74: GetUIInputData
-      75: GetUIInputModule
-      77: GetLogFilterConfig
-      79: EnableCutsceneInputMode
-      80: DisableCutsceneInputMode
-      85: EnterGPose
-      86: ExitGPose
-      87: IsInGPose
-      88: EnterIdleCam
-      89: ExitIdleCam
-      90: IsInIdleCam
-      96: ShowDeepDungeonHud
-      97: HideDeepDungeonHud
-      99: ShowEurekaHud
-      100: HideEurekaHud
-      106: OpenMycInfo
-      107: CloseMycInfo
-      108: CloseMiniMap
-      109: OpenMiniMap
-      110: IsGamePadInputActive
-      113: ToggleInputMode
-      115: IsUIHidden
-      116: AddAtkHistoryEntry
-      117: ClearAtkHistory
-      120: HandlePacket
-      121: ShowContentIntroduction
-      122: IsContentIntroductionInvisible
-      123: HideContentIntroduction
-      134: SetCursorVisibility
-      135: ToggleCursor
-      149: ShowEventFadeIn
-      150: ShowEventFadeOut
-      154: ToggleUi
-      155: ToggleUi_2
-      157: LoadScreenHideUi
-      158: LoadScreenShowUi
-      160: AnnounceHowTo
-      162: HideHowTo
-      164: ShowGoldSaucerReward
-      165: HideGoldSaucerReward
-      166: ShowTextRelicAtma
-      168: HideGoldSaucerReward_2
-      172: HideGoldSaucerReward_3
-      174: ShowHousingHarvest
-      176: OpenMiniGame
-      177: HideHousingHarvest
-      178: ShowImage
-      179: ShowText
-      180: ShowTextChain
-      181: ShowWideText
-      182: ShowPoisonText
-      183: ShowErrorText
-      184: ShowTextClassChange
-      185: ShowGetAction
-      186: ShowLocationTitle
-      187: HideLocationTitle
-      190: ShowGrandCompanyRankUp
-      193: ShowStreak
-      194: ShowAddonKillStreakForManeuvers
-      195: ShowBalloonMessage
-      196: ShowBattleTalk
-      197: ShowBattleTalkImage
-      198: ShowBattleTalkUnknown
-      199: ShowBattleTalkSound
-      201: OpenInventory
-      202: CloseInventory
-      203: IsInventoryOpen
-      204: ExecuteMainCommand
-      205: IsMainCommandUnlocked
-      208: ShowRaceCountdownEnd
-      212: IsDutyRaidFinderOpen
-      214: ShowTalkSubtitle
-      215: HideTalkSubtitle
-      218: ShowAdventureNotice
-      222: RotateLinkshellHistory
-      223: SetLinkshellCycle
-      224: RotateCrossLinkshellHistory
-      240: ShowRaceCountdownStart
-      241: ShowRaceCountdownEnd_2
+      74: GetMKDSJOB
+      75: GetMKDLORE
+      76: GetMKDSJN
+      77: GetUIInputData
+      78: GetUIInputModule
+      80: GetLogFilterConfig
+      82: EnableCutsceneInputMode
+      83: DisableCutsceneInputMode
+      88: EnterGPose
+      89: ExitGPose
+      90: IsInGPose
+      91: EnterIdleCam
+      82: ExitIdleCam
```

```suggestion
      92: ExitIdleCam
```
I'm guessing this is a typo